extern/CMakeLists.txt

```diff
@@ -1,19 +1,41 @@
-# Copyright (C) 2017-2018  iCub Facility, Istituto Italiano di Tecnologia
+# Copyright (C) 2017-2023 Fondazione Istituto Italiano di Tecnologia
 # Authors: Silvio Traversaro <silvio.traversaro@iit.it>
 # CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LICENSE
 
 # The code contained in this directory is imported directly from the src_cpp example 
-# in the MT Software Suite, version 4.8.2, available from https://www.xsens.com/mt-software-suite/
+# in the MT Software Suite, version 2022.0.0, available from https://www.xsens.com/mt-software-suite/
```

Comment:
The old URL is now broken, can you update it with the new URL from which you got the code?

Reply:
Update is https://github.com/robotology/yarp-device-xsensmt/pull/38/commits/0a495b087851691b7a989cbc3c4d0f030424db2b